Drop-In/src/main/java/com/braintreepayments/api/DropInRequest.java

```diff
@@ -68,7 +64,7 @@ public DropInRequest tokenizationKey(String tokenizationKey) {
     }
 
     /**
-     * Deprecated. Use {@link ThreeDSecureRequest#amount(String)}
+     * Deprecated. Use {@link ThreeDSecureRequest#setAmount(String)}
```

Comment:
I'm guessing y'all are leaving this for a separate PR, but we can/should remove this since it's deprecated.

Reply:
Yeah we will remove any deprecated methods in another PR 👍